assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -293,6 +338,30 @@ const baseReducer = ( state, { type, payload } ) => {
 			};
 		}
 
+		case RECEIVE_CHECK_REQUIREMENTS_ERROR: {
+			const checkRequirementsResults = { ...state.checkRequirementsResults };
+
+			for ( const [ slug, error ] of Object.entries( payload ) ) {
+				checkRequirementsResults[ slug ] = error;
+			}
```

Comment:
See above, I don't think this action needs to be able to receive multiple errors - we only need to store a single error per module.

Reply:
Updated.